evaluation/inference/human_eval/README.md

```diff
@@ -0,0 +1,45 @@
+# HumanEval Evaluation Script for DeepSpeed-FastGen
+
+## DISCLAIMER
+
+This human-eval evaluation will execute untrusted model-generated code. As per the OpenAI warning, we
+strongly recommend you sandbox your environment as described in the [human-eval paper](https://arxiv.org/pdf/2107.03374.pdf).
+
+## Setup
+
+Running the human-eval evaluation requires installation of `human_eval` with the execution code enabled,
+which requires local changes to `execution.py`. The following steps will setup `human-eval` for execution:
+
+```bash
+git clone https://github.com/openai/human-eval.git
+sed -i 's/.*#                         exec(check_program, exec_globals).*/                        exec(check_program, exec_globals)/' human-eval/human_eval/execution.py
```

Comment:
```suggestion
sed -i '/exec(check_program, exec_globals)/ s/^#//' human-eval/human_eval/execution.py
```
This might be a better/easier command

Reply:
Updated the command to use your suggestion, but added an extra space in `^#` to maintain Python spacing.